src/common/gui/main-styles.ts

```diff
@@ -2247,6 +2247,17 @@ styles.registerStyle("main", () => {
 			top: "8px",
 			left: "6px",
 		},
+		".calendar-month-week-number:after": {
```

Comment:
I think we try to avoid non-utility styles unless necessary, I would suggest to put it as a code where month number is rendered.

It's would be okay to keep it but I want to point out that this is unusual for us